docs/community/com-ingest/com-ingest-flags-sql.md

```diff
@@ -9,7 +9,7 @@ nav_order: 5
 # SQL ingest tool reference
 {: .no_toc}
 
-The SQL ingester tool selects data from a MSSQL, MySQL or Postgres database table  using a SQL endpoint.
+The SQL ingester tool selects data from a MSSQL, MySQL or Postgres database table  using a SQL CLI.
```

Comment:
Thanks for picking this up. This content was written way back in January 2023 well before FBSql was a thing.

As far as I'm aware there's 2x ways to run SQL via CLI:

FBsql > https://docs.featurebase.com/docs/tools/fbsql/fbsql-home/
SQL CLI > https://docs.featurebase.com/docs/community/com-config/com-config-sql-cli-enable/

Reply:
But to fix I've just removed "...using a SQL endpoint" because it really doesn't add anything to the explanation. I'm not sure the end user cares *how* this occurs, just that it *does* :)

Reply:
Sounds good! Looks like the `cli` subcommand was removed so we can probably remove that page:

```
jacob@jacob:~/code/featurebase$ ~/go/bin/featurebase cli
unknown command "cli" for "featurebase"
```

Reply:
has it?! No-one's told me >.<

Some of the commands need to be run from a specific directory FYI (unless you've added FeatureBase install dir to your path). Maybe give it a try there... Can't recall which directory it is. Not the one with the molecula consumer files :)